tests/PKSim.Tests/IntegrationTests/CKDDiseaseStateImplementationSpecs.cs

```diff
@@ -0,0 +1,91 @@
+   {
+      protected IDiseaseStateRepository _diseaseStateRepository;
+      protected DiseaseState _diseaseStateCKD;
+      protected IParameter _targetGFR;
+
+      public override void GlobalContext()
+      {
+         base.GlobalContext();
+         _diseaseStateRepository = IoC.Resolve<IDiseaseStateRepository>();
+         _diseaseStateCKD = _diseaseStateRepository.FindById(CoreConstants.DiseaseStates.CKD);
+         _targetGFR = _diseaseStateCKD.Parameter(CoreConstantsForSpecs.Parameter.TARGET_GFR);
+      }
+   }
+
+   public class When_applying_the_CKD_disease_state_algorithm_to_a_stage_4_individual : concern_for_CKDDiseaseStateImplementation
+   {
+      private Individual _individual;
+
+      public override void GlobalContext()
+      {
+         base.GlobalContext();
+         _individual = DomainFactoryForSpecs.CreateStandardIndividual(CoreConstants.Population.ICRP);
+         
+         //verify that the individual as the expected values
+         _individual.Age.ShouldBeEqualTo(30);
+         _individual.InputHeight.ShouldBeEqualTo(17.6);
+         _individual.InputWeight.ShouldBeEqualTo(73);
+         _individual.InputWeight.ShouldBeEqualTo(73);
+         _individual.Organism.Parameter(BMI).ValueInDisplayUnit.ShouldBeEqualTo(23.57, 1e-2);
+         _individual.Organism.Organ(FAT).Parameter(VOLUME).ValueInDisplayUnit.ShouldBeEqualTo(14.8680, 1e-2);
+         _individual.Organism.Organ(KIDNEY).Parameter(SPECIFIC_BLOOD_FLOW_RATE).ValueInDisplayUnit.ShouldBeEqualTo(302.705, 1e-2);
+         _individual.OriginData.DiseaseState = _diseaseStateCKD;
+         _individual.OriginData.AddDiseaseStateParameter(new OriginDataParameter
+         {
+            Name = _targetGFR.Name,
+            Value = _targetGFR.ConvertToBaseUnit(22, _targetGFR.DisplayUnitName()),
+            Unit = _targetGFR.DisplayUnitName(),
+         });
+
+      }
+
+      protected override void Because()
+      {
+         sut.ApplyTo(_individual);
+      }
+      
+    
+      [Observation]
+      public void should_return_the_expected_values()
+      {
+         var organism = _individual.Organism;
+         var kidney = organism.Organ(KIDNEY);
+         var fat = organism.Organ(FAT);
+         var smallIntestine = organism.Organ(SMALL_INTESTINE);
+         var hct = organism.Parameter(HCT);
+         var kidneyVolume = kidney.Parameter(VOLUME);
+         var kidneySpecificBloodFlowRate = kidney.Parameter(SPECIFIC_BLOOD_FLOW_RATE);
+         var fatVolume = fat.Parameter(VOLUME);
+         var stomach = organism.EntityAt<IContainer>(LUMEN, STOMACH);
+         var plasmaProteinScaleFactorParameter = organism.Parameter(PLASMA_PROTEIN_SCALE_FACTOR);
+         var smallIntestinalTransitTime = smallIntestine.Parameter(SMALL_INTESTINAL_TRANSIT_TIME);
+         var gastricEmptyingTime = stomach.Parameter(GASTRIC_EMPTYING_TIME);
+
+         hct.Value.ShouldBeEqualTo(0.4307472527, 1e-2);
```

Comment:
These are all values coming from a table that @prvmalik gave me (I had to adjust slightly because the base individual is slightly different)

Reply:
Did you get that note on the hematocrit fraction, about denominator being 45.5

Reply:
hum no. I am using 45.5 for healthy value for man. Is this not the right value?
![image](https://user-images.githubusercontent.com/1041237/145085699-43ea262c-807c-4e86-a678-32322ea3700f.png)

Reply:
Yes 45.5 is the correct value. You can see a typo down the right column where it is listed as 44.5 by mistake.

Reply:
oh yeah lol. Did not even realize :)